components/ShippingRates/RateTable.vue

```diff
@@ -78,7 +78,7 @@ const props = defineProps({
     type: Boolean,
     default: false
   },
-  readOnly: {
+  isClassView: {
```

Comment:
For reusability and readability, we should name props according to features/behaviour, instead of where we are using it for now
How does `isClassView` changes the rate table behaviour?